scripts/create-event-file.js

```diff
@@ -0,0 +1,495 @@
+  const presenterName = await askQuestion('👤 Presenter Name: ');
+  const presenterRole = await askQuestion('💼 Presenter Role: ');
+
+  // Get event details
+  const date = await askQuestion('📅 Date (YYYY-MM-DD): ');
+  const time = await askQuestion('⏰ Time (HH:MM): ');
+  const timezone = await askQuestion('🌍 Timezone (EDT/EST/PDT/PST): ');
+  const duration = await askQuestion('⏱️  Duration (e.g., "60 minutes"): ');
+
+  // Get tags
+  const level = await askQuestion('📊 Level (Beginner/Intermediate/Advanced): ');
+  const topics = await askQuestion('🏷️  Topics (comma-separated): ').split(',').map(t => t.trim()).filter(t => t);
+  const languages = await askQuestion('💻 Languages (comma-separated): ').split(',').map(t => t.trim()).filter(t => t);
+  const clouds = await askQuestion('☁️  Clouds (comma-separated): ').split(',').map(t => t.trim()).filter(t => t);
+
+  // Get form IDs
+  const hubspotFormId = await askQuestion('📋 HubSpot Form ID: ');
+  const salesforceCampaignId = await askQuestion('📊 Salesforce Campaign ID: ');
+
+  // Create folder and file
+  const slug = createSlug(title);
+  const folderPath = path.join('content', 'events', slug);
+  const filePath = path.join(folderPath, 'index.md');
+
+  // Create directory if it doesn't exist
+  if (!fs.existsSync(folderPath)) {
+    fs.mkdirSync(folderPath, { recursive: true });
+  }
+
+  // Generate content
+  const sortableDate = formatSortableDate(date, time, timezone);
+  const endTime = new Date(new Date(`${date}T${time}`).getTime() + (parseInt(duration) * 60000));
+  const endDate = endTime.toISOString().replace('Z', timezone === 'EDT' ? '-04:00' : 
+                                                      timezone === 'EST' ? '-05:00' :
+                                                      timezone === 'PDT' ? '-07:00' :
+                                                      timezone === 'PST' ? '-08:00' : '-00:00');
+
+  const content = `---
+# Name of the event, <= 60 characters
+title: "${title}"
+meta_desc: ${metaDesc}
+meta_image:
+
+# A featured webinar will display first in the list.
+featured: false
+
+# Webinars with unlisted as true will not be shown on the webinar list
+unlisted: false
+
+# Gated webinars will have a registration form and the user will need
+# to fill out the form before viewing.
+gated: true
+
+# The layout of the landing page.
+type: webinars
+
+# External webinars will link to an external page instead of a webinar
+# landing/registration page. If the webinar is external you will need
+# set the 'block_external_search_index' flag to true so Google does not index
+# the webinar page created.
+external: false
+block_external_search_index: false
+
+# The url slug for the webinar landing page. If this is an external
+# webinar, use the external URL as the value here.
+url_slug: ${slug}
+
+# Content for the left hand side section of the page.
+main:
+    # Webinar title.
+    title: "${title}"
+    event_type: workshop # workshop | event
```

Comment:
if we want this to work for in-person (which it easily could) this should be var'd